pkgs/applications/video/vlc/default.nix

```diff
@@ -52,6 +54,7 @@ stdenv.mkDerivation rec {
     fluidsynth wayland wayland-protocols
   ] ++ optional (!stdenv.hostPlatform.isAarch64) live555
     ++ optionals withQt5    [ qtbase qtsvg qtx11extras ]
+    ++ optionals skins2Support (with xorg; [ libXpm freetype libXext libXinerama ])
```

Comment:
```suggestion
    ++ optionals skins2Support [ libXpm freetype libXext libXinerama ]
```

Reply:
Is `xorg` deprecated? I asked @lucasew to use `xorg` because it is already used in https://github.com/NixOS/nixpkgs/pull/109235/files#diff-e0d4a136479523bc5cfaf6a3bf53dcdb52dd51911f7fbaf199a0136e8e49371fR52.

If the idea is to not used `xorg`, I would prefer to not use it on https://github.com/NixOS/nixpkgs/pull/109235/files#diff-e0d4a136479523bc5cfaf6a3bf53dcdb52dd51911f7fbaf199a0136e8e49371fR52.

Reply:
But I am asking it because I don't want to confuse the users when doing reviews.

Reply:
GitHub didn't show this anymore so I didn't see the discussion.

not using xorg requires an inherit so this is fine.

Reply:
So this issue is solved them?